simplex/block_test.go

```diff
@@ -0,0 +1,149 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package simplex
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/StephenButtolph/canoto"
+	"github.com/ava-labs/simplex"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ava-labs/avalanchego/snow/consensus/snowman"
+	"github.com/ava-labs/avalanchego/snow/consensus/snowman/snowmantest"
+	"github.com/ava-labs/avalanchego/snow/engine/enginetest"
+	"github.com/ava-labs/avalanchego/snow/engine/snowman/block/blocktest"
+)
+
+func TestBlockSerialization(t *testing.T) {
+	unexpectedBlockBytes := errors.New("unexpected block bytes")
+
+	testVM := &blocktest.VM{
+		VM: enginetest.VM{
+			T: t,
+		},
+	}
```

Comment:
`testVM` is defined outside the test loop, but it is not used in setting up the vectors. Can we define this struct inside the test loop?